koku/masu/processor/tasks.py

```diff
@@ -352,6 +352,9 @@ def update_cost_model_costs(schema_name, provider_uuid, start_date=None, end_dat
         updater = CostModelCostUpdater(schema_name, provider_uuid)
         if updater:
             updater.update_cost_model_costs(start_date, end_date)
+            if provider_type:
+                refresh_materialized_views.delay(schema_name, provider_type)
```

Comment:
Since this is already being called in a worker task.  I think we could remove this `delay` to run it synchronously.

Reply:
Removed the `delay`.